ext-src/swoole_http_request.cc

```diff
@@ -102,68 +103,7 @@ static inline char *http_trim_double_quote(char *ptr, int *len) {
     return tmp;
 }
 
-static sw_inline const char *http_get_method_name(int method) {
-    switch (method) {
-    case PHP_HTTP_GET:
-        return "GET";
-    case PHP_HTTP_POST:
-        return "POST";
-    case PHP_HTTP_HEAD:
-        return "HEAD";
```

Comment:
Perhaps we could simply change the contents of this function to

```cpp
 return swoole_http_method_str(method)
```

Reply:
done